src/Appwrite/Platform/Workers/Migrations.php

```
@@ -103,7 +104,8 @@ public function action(
         Device $deviceForMigrations,
         Device $deviceForFiles,
         Mail $queueForMails,
-        StatsUsage $queueForStatsUsage,
+        Context $usage,
+        \Appwrite\Event\Publisher\Usage $publisherForUsage,
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$publisherForUsage'. (undefined)

(UnusedFormalParameter)